client/my-sites/checkout/controller.jsx

```diff
@@ -70,6 +71,19 @@ export function checkout( context, next ) {
 		context.pathname.includes( '/checkout/no-site' ) &&
 		'no-user' === context.query.cart;
 
+	const searchParams = new URLSearchParams( window.location.search );
+	const isSignupCheckout = searchParams.get( 'signup' ) === '1';
```

Comment:
Could you help me point out where this is set in the first place? I'd like to confirm how confident we can rely on this.

Reply:
It is being set in [this line](https://github.com/Automattic/wp-calypso/blob/master/client/signup/config/flows.js#L26).